docs/src/testnets/internal-testnet-1/README.md

```diff
@@ -1,35 +1,78 @@
 # Internal Testnet 1
 
-*Last updated on **3/30/2022** by **Alexandre Roque***
+##### Last updated on 04/07/2022 by Alexandre Roque
 
-Current chain id `anoma-masp-0.3.51d2f83a8412b95` and branch
-`internal/testnet-n1` (commit
-`0184e64e044366ec370d1431ddf4691b4bd3a5b4`)
+>⚠️ the values below might change frequently.
+>
+>Latest values regarding the testnet that would be useful to have in your shell:
+>
+>```bash
+>export ANOMA_TESTNET_CHAIN_ID='anoma-masp-0.3.51d2f83a8412b95'
```

Comment:
For this page we will now only need `export ANOMA_TESTNET_CHAIN_ID` although we should still mention the branch/commit information (in the `## Installing Namada`) section